.github/workflows/ci.yml

```diff
@@ -43,43 +48,65 @@ jobs:
           - honeycomb_cover
     steps:
       - name: Set up Git repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
+
       - name: Inject environment variables
         run: |
           sed -i env.tex -e "s/GITHUBCOMMITHASH/${GITHUB_SHA:0:6}/g"
           sed -i env.tex -e "s/GITHUBRUNNUMBER/${GITHUB_RUN_ID}/g"
           cat env.tex
+
       - name: Compile LaTeX document
         uses: xu-cheng/latex-action@v2
         with:
           root_file: ${{ matrix.document }}.tex
+
       - name: Upload document
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-pages-artifact@v1
```

Comment:
This probably should not have changed. Each pdf is built and uploaded separately as an artifact, then downloaded all back in a single downstream job that will tar them all up.

Reply:
Fixing in #2.